tools/complex_permissions_test.py

```diff
@@ -92,6 +94,13 @@ def test(self):
                       self.test_outside_test_and_js_dir, test_type)
             wrap_test(test_name_base + "_inside_tests_and_js_dir",
                       self.test_inside_test_and_js_dir, test_type)
+            wrap_test(test_name_base + "_relative", self.test_relative,
+                      test_type)
+            wrap_test(test_name_base + "_no_prefix", self.test_no_prefix,
+                      test_type)
+            if os.name != "nt":
```

Comment:
Not sure if this is needed, since this test isn't run on windows anyways. https://github.com/denoland/deno/blob/2edee3367dc9003b721cf87ca57e820c7acf7b25/tools/test.py#L105-L111